src/sdk/index.js

```diff
@@ -4,23 +4,42 @@ import {merge} from 'lodash'
 import api from './api'
 import {DEFAULT_HEADERS} from './headers'
 
+function configGlobalSpec (globalSpec) {
+
+  let headers
+  if (typeof globalSpec.headers === 'function') {
+    headers = {...globalSpec.headers()}
```

Comment:
Ah ok so it's `origin` not `x-entity` or `auth`:
`globalSpec = {origin: 'string': headers: {x-entity: 'string', authorization: 'string'}}`